docs/sources/enterprise/license-expiration.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "License Expiration"
+description = ""
+keywords = ["grafana", "licensing"]
+type = "docs"
+[menu.docs]
+parent = "enterprise"
+weight = 8
++++
+
+# Expired License
+
+If your license has expired, your Grafana admin needs to upload a new *license.jwt* file to the location in the `enterprise.license_path` directory (default your Grafana data directory). This file can be downloaded from your Grafana Cloud organization’s license page. Restart Grafana once the file is in place.
```

Comment:
```suggestion
If your license has expired, then your Grafana admin needs to upload a new *license.jwt* file to the location in the `enterprise.license_path` directory (default is your Grafana data directory). This file can be downloaded from your Grafana Cloud organization’s license page. Restart Grafana once the file is in place.
```

Reply:
This paragraph should actually be a task called Renew your license, organized as numbered steps.